submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                  <option value="Arabian" selected>Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian">Ukrainian</option>
+                </select>
+              </div>
+              <div class="select-button select-button-next">
+                <select name="language" class="select-button-inner" required>
+                  <option value="english">English</option>
+                  <option value="Arabian">Arabian</option>
+                  <option value="Kudrish">Kudrish</option>
+                  <option value="Ukrainian" selected>Ukrainian</option>
+                </select>
+              </div>
+              <img
+                tabindex="0"
+                src="./img/waste-bin.svg"
+                class="deleteFile select-delete"
+              />
+              <button type="button" class="add-button">
+                Add
+              </button>
+            </div>
+          </section>
+          <section class="toneSelect">
+            <div class="allTextDocumentUpload">
+              <span class="numberSection"><b>3</b></span>
+              <h2 class="uploadText">Select your desired tone</h2>
+              <span class="numberSection second-elem-numb"><b>4</b></span>
+              <h2 class="uploadText second-elem">
+                Tell us how urgent this job is
+              </h2>
+            </div>
+            <div class="tone-time-container">
+              <div class="tone">
+                <label for="friendly">
+                  <input type="radio" name="tone" id="friendly" class="input" />
+                  <div class="select-tone tone-before chat">
+                    <span class="checkedTone"></span>Friendly / Casual
+                  </div>
+                </label>
+                <label for="corporate">
+                  <input
+                    type="radio"
+                    name="tone"
+                    id="corporate"
+                    class="input"
+                    checked
+                  />
+                  <div class="select-tone tone-before suitcase">
+                    <span class="checkedTone"></span>Corporate / Pro
+                  </div>
+                </label>
+                <label for="media">
+                  <input type="radio" name="tone" id="media" class="input" />
+                  <div class="select-tone tone-before media">
+                    <span class="checkedTone"></span>Media / Formal
+                  </div>
+                </label>
+              </div>
+              <div class="time-range">
+                <input
+                  type="range"
+                  class="time-range-button"
+                  min="0"
+                  max="100"
+                  step="1"
+                  value="50"
+                />
+                <div class="range-text">
+                  <h5 class="range-under">I GOT TIME</h5>
```

Comment:
In this case, it's just text, not heading, better to use `span` or `p` for this text.